FhirToDataLake/scripts/Set-SynapseEnvironment.ps1

```diff
@@ -40,19 +42,22 @@ Param(
     [string]$ResultPath = "result",
     [string]$SqlScriptCollectionPath = "sql/Resources",
     [string]$MasterKey = "FhirSynapseLink0!",
-    [int]$Concurrent = 30
+    [int]$Concurrent = 30,
+    [string]$CustomizedSchemaImage
 )
 
-$jobName = "FhirSynapseJob"
-$readmePath = ".readme.txt"
+$JobName = "FhirSynapseJob"
+$PlaceHolderName = ".readme.txt"
+$OrasAppPath = "oras.exe"
```

Comment:
Maybe we can have a oras link and download oras to a specific path when initializing so that we can avoid committing an executable.

Reply:
Updated, download oras application in script, thanks.